From 3255945644ba5b69865ac2a0164dd4fd0ce5405e Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Sat, 25 Aug 2018 01:02:00 +0100
Subject: [PATCH 06/11] AppOpsManager: Update with the new ops

 * Includes new ops introduced in Pie as well as ours

Change-Id: I713bbdee1da9d142cf9671128fa05391fa38c48a
---
 libs/binder/include/binder/AppOpsManager.h | 25 ++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/libs/binder/include/binder/AppOpsManager.h b/libs/binder/include/binder/AppOpsManager.h
index c5b57c7ed..fb682ecde 100644
--- a/libs/binder/include/binder/AppOpsManager.h
+++ b/libs/binder/include/binder/AppOpsManager.h
@@ -94,7 +94,32 @@ public:
         OP_PROCESS_OUTGOING_CALLS = 54,
         OP_USE_FINGERPRINT = 55,
         OP_BODY_SENSORS = 56,
+        OP_READ_CELL_BROADCASTS = 57,
+        OP_MOCK_LOCATION = 58,
+        OP_READ_EXTERNAL_STORAGE = 59,
+        OP_WRITE_EXTERNAL_STORAGE = 60,
+        OP_TURN_SCREEN_ON = 61,
+        OP_GET_ACCOUNTS = 62,
+        OP_RUN_IN_BACKGROUND = 63,
         OP_AUDIO_ACCESSIBILITY_VOLUME = 64,
+        OP_READ_PHONE_NUMBERS = 65,
+        OP_REQUEST_INSTALL_PACKAGES = 66,
+        OP_PICTURE_IN_PICTURE = 67,
+        OP_INSTANT_APP_START_FOREGROUND = 68,
+        OP_ANSWER_PHONE_CALLS = 69,
+        OP_RUN_ANY_IN_BACKGROUND = 70,
+        OP_CHANGE_WIFI_STATE = 71,
+        OP_REQUEST_DELETE_PACKAGES = 72,
+        OP_BIND_ACCESSIBILITY_SERVICE = 73,
+        OP_ACCEPT_HANDOVER = 74,
+        OP_MANAGE_IPSEC_TUNNELS = 75,
+        OP_START_FOREGROUND = 76,
+        OP_BLUETOOTH_SCAN = 77,
+        OP_BLUETOOTH_CHANGE = 78,
+        OP_BOOT_COMPLETED = 79,
+        OP_NFC_CHANGE = 80,
+        OP_DATA_CONNECT_CHANGE = 81,
+        OP_SU = 82
     };
 
     AppOpsManager();
-- 
2.17.1

